ci/tests.py

```diff
@@ -30,13 +31,17 @@
 
 
 def _availability_zones():
```

Comment:
make  this method a static property of class tempest manager. use caching, so that the output can be reused during the same run.